app/config/AppConfig.scala

```diff
@@ -37,5 +37,7 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
   val eisgbUrl: String = eisBaseUrl ++ config.get[String]("microservice.services.eis.gb.uri")
   val eisgbBearerToken: String =
     config.get[String]("microservice.services.eis.gb.headers.bearerToken")
+  val nctsBaseUrl: String = servicesConfig.baseUrl("ncts")
```

Comment:
Can we call this something more specific than just `ncts`? NCTS means a lot of different things in different contexts. Perhaps `ntcs-monitoring`?

Reply:
I don't mind doing that within this service, but to rename all of our repos will be a bit of a pain.